### PR TITLE
Vectorize `is_installed()`

### PR DESCRIPTION
**Pull request by salim-b**
I'd like to have a vectorized `is_installed()` so I just went ahead and created this PR.

### CODE REVIEW

**Comment by lionel-** [comment]
I think all `is_` functions should return a single non-missing `TRUE` or `FALSE` value, i.e. they should be safe to use in `if ()` conditions.

**Comment by salim-b** [comment]
Ok. But before this change, `is_installed()` silently tested the first element of the `pkg` argument only. So it always returned `TRUE` if the _first_ package was installed regardless of the remaining ones:

```r
rlang::is_installed(pkg = c("base", "no.notarealpackage"))
#> [1] TRUE
```

<sup>Created on 2020-06-08 by the <a href="https://reprex.tidyverse.org">reprex package</a> (v0.3.0)</sup>

Would it be fine to reduce the return value to a logical scalar using `all()`?

**Comment by lionel-** [comment]
I think that makes sense.

File: R/env-special.R [hunk 1]
```diff
@@ -264,19 +264,23 @@ is_namespace <- function(x) {
...
+#'   installed package and `FALSE` for each package not found in any of the
+#'   libraries.
 #' @export
 #' @examples
-#' is_installed("utils")
+#' is_installed(c("graphics", "utils"))
 #' is_installed("ggplot5")
 is_installed <- function(pkg) {
-  is_true(requireNamespace(pkg, quietly = TRUE))
+  vapply(pkg,
```
Comment (lionel-) [inline R/env-special.R:281]: Can you use `map_lgl()` and follow the local style please?

Comment (salim-b) [reply R/env-special.R]: done

**Comment by salim-b** [comment]
PR updated :)

**Comment by lionel-** [comment]
Can you also add a NEWS bullet with your handle please?

**Comment by salim-b** [comment]
> Can you also add a NEWS bullet with your handle please?

Ok, done.

**Comment by lionel-** [comment]
Thanks!

**Pull request merged**